libs/comments/website/src/lib/comment/comment.tsx

```diff
@@ -146,7 +151,14 @@ export const Comment = ({
             )}
           </CommentName>
 
-          {flair && <CommentFlair isGuest={isGuest}>{flair}</CommentFlair>}
+          {source && (
```

Comment:
This now shows both the flair & the source, is that intended?

Reply:
fixed